packages/block-editor/src/utils/get-font-styles-and-weights.js

```diff
@@ -118,7 +125,8 @@ export function getFontStylesAndWeights( fontFamilyFaces ) {
 				fontStyles.push( fontStyle );
 			}
 		}
-		if ( fontWeight ) {
+
+		if ( Object.keys( fontWeight ).length ) {
```

Comment:
I think this change is great. What do you think to also updating the `if ( fontStyle )` check, on line 119, so that these checks are consistent?

Reply:
Thanks @mikachan ! I've added the check here https://github.com/WordPress/gutenberg/pull/64953/commits/ca110e5d27710e7d850d4146e9f4133cc6d8ea9a